pkgs/kde/misc/calligra/default.nix

```diff
@@ -0,0 +1,82 @@
+{
+  mkKdeDerivation,
+  lib,
+  fetchurl,
+  boost,
+  eigen,
+  imath,
+  libetonyek,
+  libgit2,
+  libodfgen,
+  librevenge,
+  libvisio,
+  libwpd,
+  libwpg,
+  libwps,
+  okular,
+  perl,
+  pkg-config,
+  poppler,
+  qtkeychain,
+  qtsvg,
+}:
+
+mkKdeDerivation rec {
+  pname = "calligra";
+  version = "4.0.1";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/calligra/calligra-${version}.tar.xz";
+    hash = "sha256-1AH15z9PG9wLNUjqGlCwrBd4we3jCmozWUTtf72I2V8=";
+  };
+
+  extraBuildInputs = [
+    # Required for any build.
+    boost
+    qtkeychain
+    qtsvg
+
+    # Required for Calligra Sheets.
+    eigen
+    imath
+
+    libgit2 # Git integration.
+    okular.dev # Okular integration.
```

Comment:
Shouldn't need explicit `.dev` on inputs, also comments are a bit too much IMO but not a blocker